### PR TITLE
Function UI redesign

### PR DESCRIPTION
**Pull request by SvetaMemphis**

### CODE REVIEW

File: ui_src/src/domain/functions/components/functionBox/index.js [hunk 1]
```diff
@@ -57,71 +63,77 @@ function FunctionBox({ funcDetails }) {
...
         setOpen(flag);
-        // if (flag) {
-        //     history.push(`${pathDomains.functions}/${functionDetails?.function_name}`);
-        //     setSelectedFunction(functionDetails?.function_name);
-        // } else {
-        //     history.push(`${pathDomains.functions}`);
-        //     setSelectedFunction('');
-        // }
+        if (flag) {
+            // history.push(`${pathDomains.functions}/${functionDetails?.function_name}`);
```
Comment (idanasulinmemphis) [inline ui_src/src/domain/functions/components/functionBox/index.js:67]: remove comments if they are not needed

File: ui_src/src/domain/functions/components/functionBox/index.js [hunk 2]
```diff
@@ -57,71 +63,77 @@ function FunctionBox({ funcDetails }) {
...
+            setSelectedFunction(functionDetails);
+        } else {
+            // history.push(`${pathDomains.functions}`);
```
Comment (idanasulinmemphis) [inline ui_src/src/domain/functions/components/functionBox/index.js:70]: remove comments if they are not needed

File: ui_src/src/domain/functions/components/functionBox/index.js [hunk 3]
```diff
@@ -57,71 +63,77 @@ function FunctionBox({ funcDetails }) {
...
+                                width="100px"
+                                height="34px"
+                                placeholder={'Install'}
+                                colorType="white"
+                                radiusType="circle"
+                                backgroundColorType="purple"
+                                fontSize="12px"
+                                fontFamily="InterSemiBold"
+                                disabled={(isCloud() && !integrated) || !isCloud()}
+                                // isLoading={loader}
```
Comment (idanasulinmemphis) [inline ui_src/src/domain/functions/components/functionBox/index.js:124]: do we need it?

File: ui_src/src/domain/functions/components/functionBox/style.scss [hunk 4]
```diff
@@ -1,80 +1,91 @@
...
-        border-bottom: 1px solid #f5f5f5;
+        margin-top: 5px;
+        padding: 10px 16px;
+
+        border-top: 1px solid #f5f5f5;
         overflow-x: auto;
-        .tag-wrapper {
-            max-width: 95px !important;
-        }
+        // .tag-wrapper {
```
Comment (idanasulinmemphis) [inline ui_src/src/domain/functions/components/functionBox/style.scss:86]: remove if not needed

File: ui_src/src/domain/functions/components/functionList/style.scss [hunk 5]
```diff
@@ -48,12 +48,12 @@
 
         .cards-wrapper {
             width: 100%;
-            height: 100%;
-            display: grid;
-            grid-template-columns: repeat(auto-fill, 320px);
-            grid-auto-rows: 210px;
-            gap: 15px;
-            align-items: center;
+            // height: 100%;
```
Comment (idanasulinmemphis) [inline ui_src/src/domain/functions/components/functionList/style.scss:51]: remove if not needed

File: ui_src/src/domain/functions/components/functionBox/index.js (hunk 2 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/functions/components/functionBox/index.js:71]: done

File: ui_src/src/domain/functions/components/functionBox/index.js (hunk 1 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/functions/components/functionBox/index.js:68]: Done

File: ui_src/src/domain/functions/components/functionBox/index.js (hunk 3 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/functions/components/functionBox/index.js:143]: I set it to false, shouldn't we put a loader on this async task?

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**